pybikes/data/callabike.json

```diff
@@ -3,10 +3,10 @@
         {
             "tag": "callabike-aschaffenburg",
             "meta": {
-                "latitude": 49.436467,
```

Comment:
That's exactly the problem. There are three options:

1. As @piotrowski commented, an incomplete version being better than none
2. Splice the space for each network into a grid of coordinates, involving multiple calls. There's an API call to get the boundaries of each system (see https://github.com/eskerda/pybikes/issues/238#issuecomment-292649216). Together with some utility functions, the grid can be optimized to reduce the number of calls.
3. Wait until they provide a better version of their API. Contact them and complain. There's no reason for this data to not be provided cleanly.